examples/pytorch/sampling/multi-gpu/README.md

```diff
@@ -0,0 +1,50 @@
+Multi-GPU training by graph sampling
+===
+
+The example trains GNN models on a single machine but multiple GPUs by sampling from a single graph data. The setting is suitable for the case where the graph is too large to fit in one GPU memory.
```

Comment:
The graph is in the CPU memory. The major benefit of using multiple gpus is to speed up running.

Reply:
OK, to be more specific:
1. sampling is designed for the case where graph is too large to fit in one GPU memory.
2. multi-gpu setting is for speedup training.